matminer/featurizers/structure.py

```diff
@@ -3612,3 +3613,189 @@ def citations(self):
     def implementors(self):
         return ["Alex Dunn", "Lauri Himanen"]
 
+    
+class GlobalInstabilityIndex(BaseFeaturizer):
+    '''
```

Comment:
Could you change these to double rather than single quotes? (Python style docs recommend double)